packages/ember-htmlbars/lib/system/build-component-template.js

```diff
@@ -186,7 +186,7 @@ function normalizeComponentAttributes(component, attrs) {
     }
   }
 
-  normalized.role = buildStatement('get', 'ariaRole');
+  normalized.role = normalized.role || buildStatement('get', 'ariaRole');
```

Comment:
I think you can remove the `||` by moving the original line up above line 160. That way we always set `normalized.role` and only if you happen to add an attributeBinding for it would it override (on line 185).  

Can you test to see if that fixes the failing test?

Reply:
I'm getting a failure of the ability to set `ariaRole` in v 3 beta 1.
See here, all tests are passing on Travis apart from one
https://travis-ci.org/zestia/ember-select-box/jobs/352537510

Should I be using `ariaRole`, or `role`